examples/set-labels-full-coverage/README.md

```diff
@@ -34,25 +34,29 @@ apiVersion: fn.kpt.dev/v1alpha1
 kind: SetLabels
 metadata:
   name: my-config
+  annotations:
+    config.kubernetes.io/local-config: "true"
 labels:
   color: orange
   fruit: apple
+  app: new
 ```
 
 The desired labels is provided using `labels` field. We have a CRD with group
-`batch` and kind `CronJob`. 
+`dev.example.com`, version `v1` and kind `MyResource`. 
 
 ### Function invocation
 
 Invoke the function by running the following commands:
 
 ```shell
-$ kpt fn render set-labels-cronjob
+$ kpt fn render set-labels-full-coverage
 ```
 
 ### Expected result
 
-Check all resources have 2 labels: `color: orange` and `fruit: apple`. 
-The `CronJob` should have new labels in `spec.jobTemplate.metadata.labels` and `"spec.jobTemplate.spec.template.metadata.labels"` as well.
+Check all resources, if the labels to the path should be upsert, 
+then we can see 3 labels have been updated, `color: orange`, `fruit: apple` and `app: new`. 
+If the path does not need to create, just update labels if the key already exists, otherwise, ignore. 
```

Comment:
How about "The following labels should be upserted to the `labels`, `matchingLabels` or `seletors`(labelSelector) fields: ..." .

Reasons: try to avoid describing the code logic, but explaining from the usage perspective.